spyder/plugins/ipythonconsole/utils/messagehandler.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+import logging
+logger = logging.getLogger(__name__)
+
+class SpyderMessageHandler:
+    """Message handler for spyder messages. Each shell widget has one instance of it to
```

Comment:
Docstrings should maintain the correct indent level and stay consistent both with our agreed convention for multi-line docstrings (as mentioned in the wiki page linked in the PR template) and with the one used elsewhere in this PR. Try this instead:

```
"""
Message handler for Spyder messages.

Each shell widget has one instance of it to...
[...]
"""
```